onnxruntime/core/optimizer/initializer.cc

```diff
@@ -303,24 +303,34 @@ struct ScaleByAxis {
       }
     } else {
       for (size_t block_offset = 0, i = 0; i < num_blocks; i++) {
-        const auto numeric_scaler = to_numeric(scalers_data[i]);
-        for (size_t j = 0; j < block_size; ++j, ++block_offset) {
-          dst[block_offset] = T(to_numeric(dst[block_offset]) * numeric_scaler);
+        if (column_major)
+        {
```

Comment:
curly brace consistency. addressing most of the cpplint issues on this PR is probably a good way to make sure that the file has the right formatting.